### PR TITLE
WP-93 and WP-98 unit testing on Header and MoviePage  

### PR DESCRIPTION
**Pull request by G-Andrey**
ive completely lost the plot with testing

### CODE REVIEW

**Comment by PredatorFeesh** [comment]
Update: Enzyme could be a possible fix

**Comment by G-Andrey** [comment]
Added unit testing for MoviePage component by supplying it a dummy IMDb api call as well rendering tests. I've been running into way too many issues in testing the Header component so I just left it at basic rendering tests for the searchbox and search button.

File: WatchParty/client/package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "@testing-library/user-event": "^7.2.1",
     "bootstrap": "^4.4.1",
     "history": "^4.10.1",
+    "prop-types": "^15.7.2",
```
Comment (PredatorFeesh) [inline WatchParty/client/package.json:17]: :1st_place_medal: :1st_place_medal:

File: WatchParty/client/src/components/Header/__test__/Header.test.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { BrowserRouter as Router } from "react-router-dom";
+import { mount } from 'enzyme';
+
+import Header from '../Header';
+
+describe("search field and search button rendered properly", () => {
```
Comment (PredatorFeesh) [inline WatchParty/client/src/components/Header/__test__/Header.test.jsx:7]: This makes me happy :person_fencing:

**Review by PredatorFeesh — APPROVED** [review]
Get this work of art in ASAP

**Review by amckoy42 — APPROVED** [review]
Everything here looks great. Nice job learning this all so quickly!

**Pull request merged**